examples/mining_ind.py

```diff
@@ -0,0 +1,39 @@
+import desbordante
+
+def print_table(filename: str):
+    with open(filename) as table:
+        rows = [row.strip().split(',') for row in table.readlines()]
```

Comment:
Should probably use the [csv](https://docs.python.org/3/library/csv.html) module for this